onnxruntime/test/perftest/test_configuration.h

```diff
@@ -37,8 +37,8 @@ struct MachineConfig {
 
 struct RunConfig {
   std::basic_string<ORTCHAR_T> profile_file;
-  TestMode test_mode{TestMode::kFixDurationMode};
-  size_t repeated_times{1000};
+  TestMode test_mode{TestMode::KFixRepeatedTimesMode};
+  size_t repeated_times{15};
```

Comment:
this default value is explicitly mentioned in the help text:

https://github.com/microsoft/onnxruntime/blob/0b235b27632e96c5f9cf01093d9d4d4ecf2f8447/onnxruntime/test/perftest/command_args_parser.cc#L40

also, not sure if there are usages out there that rely on the current defaults. those would start behaving differently.

Reply:
Good catch, thanks. Fixed.

Reply:
Actually, I want to refactor the meaning of arg `-I`. It means 'use generated random data' now, how about changing to " -I /xx/x/input_data_path". It seems more reasonable.

Reply:
I think the default now is to get the input data from a path relative to the model, which doesn't seem unreasonable to me. I don't know if we care about preserving the existing default behavior of the options at all. @skottmckay @snnn Any thoughts here?

Also, I didn't realize the `-I` behavior changed to generate random data for certain types (missed looking at that PR). We should probably provide a way to configure the range of generated values too.

Reply:
Let's keep the default `-I` behavior unchanged first.

Reply:
Input data from path relative to the model is consistent with how onnx_test_runner works so it's better to preserve that.

Arbitrarily changing the default test mode may cause unexpected consequences though. e.g. there may be perf tests in CIs that don't explicitly set the mode. 

Maybe as an alternative we could get rid of the unnecessary `-m` argument for mode as it can be inferred from whether `-r` or `-d` is specified and add a check that one of those two is provided. No defaults are required in that case and any existing test that relied on them should break (and can be fixed). Defaults are fairly meaningless anyway - how long or how many iterations is highly dependent on the model and hardware.